adapters/richaudience/richaudience.go

```diff
@@ -0,0 +1,351 @@
+package richaudience
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type RichaudienceAdapter struct {
```

Comment:
Please rename this to just `adapter` instead of `RichaudienceAdapter`

Reply:
in next commit i have solved: 

I have change name of struct in adapter